webhook/deploy.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# https://github.com/DevelopingSpace/starchart/pkgs/container/starchart
+IMAGE=ghcr.io/developingspace/starchart
+SERVICE=starchart_mycustomdomain
+
+# We expect to receive the new image tag via the TAG environment variable
+if [[ -z "$TAG" ]]; then
```

Comment:
Idk how to read this, what is -z?

Reply:
Seems like this is to check it $TAG is null